modules/commons/src/main/frontend/src/SearchBar.jsx

```diff
@@ -57,6 +53,7 @@ function SearchBar(props) {
   const { classes, className, defaultValue, invertColors, onChange, onPopperClose, onSelect, onSelectFinish, queryConstructor, resultConstructor, staticContext, ...rest } = props;
   const [ search, setSearch ] = useState(defaultValue);
   const [ results, setResults ] = useState([]);
+  const [ moreResults, setMoreResults ] = useState([]);
```

Comment:
We set a number later, and we check its value with `> 0`, so I think this is more correct:

```suggestion
  const [ moreResults, setMoreResults ] = useState(0);
```